### PR TITLE
Update DG for v1.2

### LINKED ISSUE #31: Update DG
- Update use cases to support multiple fields in a one-shot command
- Add the 'save' feature explicitly under user stories (high-priority)

### PR DESCRIPTION
**Pull request by ckcherry23**
- Use cases now capture that multiple fields can be edited at once
- Added 'save data' features (saving automatically and locally) as high-priority user stories

Closes #31

### CODE REVIEW

**Review by NicsunXnus — APPROVED** [review]
lgtm

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -279,7 +279,7 @@ We help _busy working professionals_ manage their large list of contacts by prov
 Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unlikely to have) - `*`
 
 | Priority | As a …​              | I want to …​                                                     | So that I can…​                                                                    |
-| -------- | ----------------------- | ------------------------------------------------------------------- | ------------------------------------------------------------------------------------- |
+| -------- |-------------------------|---------------------------------------------------------------------|---------------------------------------------------------------------------------------|
```
Comment (NicsunXnus) [inline docs/DeveloperGuide.md:282]: Does this changes work for the website?

Comment (ckcherry23) [reply docs/DeveloperGuide.md:282]: I won't be able to test until it is deployed. Will revert just in case.

Comment (ckcherry23) [inline docs/DeveloperGuide.md:282]:
```suggestion
| -------- | ----------------------- | ------------------------------------------------------------------- | ------------------------------------------------------------------------------------- |
```

**Pull request merged**